diff --git a/cjs/react-dom-client.development.js b/cjs/react-dom-client.development.js
index fa853b3a79413c8ddc553441ee259d4b34f59d69..60be2b42bead0e9ce550c6d5ed6c182c26bd7f2c 100644
--- a/cjs/react-dom-client.development.js
+++ b/cjs/react-dom-client.development.js
@@ -6008,17 +6008,7 @@
       }
     }
     function resolveLazy(lazyType) {
-      try {
-        return callLazyInitInDEV(lazyType);
-      } catch (x) {
-        if (null !== x && "object" === typeof x && "function" === typeof x.then)
-          throw (
-            ((suspendedThenable = x),
-            (needsToResetSuspendedThenableDEV = !0),
-            SuspenseException)
-          );
-        throw x;
-      }
+      return callLazyInitInDEV(lazyType);
     }
     function getSuspendedThenable() {
       if (null === suspendedThenable)
